java/src/org/openqa/selenium/chrome/ChromeDriver.java

```diff
@@ -90,8 +96,12 @@ public static RemoteWebDriverBuilder builder() {
   }
 
   private static class ChromeDriverCommandExecutor extends ChromiumDriverCommandExecutor {
-    public ChromeDriverCommandExecutor(DriverService service) {
-      super(service, getExtraCommands());
+    public ChromeDriverCommandExecutor(DriverService service) throws URISyntaxException {
```

Comment:
Please remove the `throws clause` here and also kindly help clean up the same from all the constructors as well. It's not needed.

Reply:
Also I think we can remove this now un-used constructor